src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -66,6 +66,24 @@ public function testInvalidCwd()
         $cmd->run();
     }
 
+    public function testInvalidCommand()
+    {
+        $cmds = [
+            new Process(['invalid']),
+            Process::fromShellCommandline('invalid'),
+        ];
```

Comment:
What about using a data provider?

Reply:
This would indeed be better with a data provider or 2 tests rather than a loop inside the test emulating having multiple tests

Reply:
Done in d603793013e2294d8f97c378b0256e86c8dfedb9.